libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackBottomSheet.java

```diff
@@ -57,10 +64,23 @@ public static FeedbackBottomSheet newInstance(FeedbackBottomSheetListener feedba
     return feedbackBottomSheet;
   }
 
+  public static FeedbackBottomSheet newInstance(FeedbackBottomSheetListener feedbackBottomSheetListener,
```

Comment:
For documentation purposed noting that probably this needs to be added to UI SDK migration docs cc @langsmith